lib/matplotlib/axes/_axes.py

```diff
@@ -55,6 +55,11 @@ class Axes(_AxesBase):
     are 'xlim_changed' and 'ylim_changed' and the callback will be called with
     func(*ax*) where *ax* is the `Axes` instance.
 
+    .. note::
+
+        As a user, you do not instantiate Axes directly, but use Axes creation
+        methods instead.
```

Comment:
Can we give a couple of examples here?  

```
e.g. From `.pyplot` or `.figure`: `figure.add_axes`, `figure.subplots`, `figure.subplot_mosaic`.
```

Reply:
Ideally, we'd have https://github.com/matplotlib/matplotlib/issues/17376 to link to. But since this does not exist yet, I've taken the suggestion to give some examples.